libraries/provider_repo.rb

```diff
@@ -27,6 +27,7 @@ class NetuitiveCookbook::NetuitiveRepoProvider < Chef::Provider::LWRPBase
       include_recipe 'apt'
       # add repo and key
       apt_repository 'netuitive' do
+        distribution ''
```

Comment:
I think we want to make this an attribute and have a check to determine if we want this. I am hesitant to hardcode things.